src/views/Admin/RepaymentSchedule.vue

```diff
@@ -0,0 +1,410 @@
+				<div class="tab-content" id="tabContent">
+					<div class="tab-pane fade show active" id="all" role="tabpanel" aria-labelledby="all-tab">
+						<repayment-table :customers="orders" :OId="OId"></repayment-table>
+					</div>
+
+				</div>
+			</div>
+
+			<div v-else class="mx-4">
+				<zero-state :title="'No Repayment List'" :message="'There are currrently no customer that is owing'">
+					<template v-slot:image>
+						<img src="../../assets/thumb-up.png" />
+					</template>
+				</zero-state>
+			</div>
+		</div>
+
+		<div v-if="pageParams && orders.length > 0">
+			<base-pagination :page-param="pageParams" @fetchData="getReport">
+			</base-pagination>
+		</div>
+
+	</div>
+</template>
+<script>
+import { get, byMethod } from '../../utilities/api';
+import Flash from '../../utilities/flash';
+import StatCard from '../../components/StatCard.vue';
+import users from '../../assets/users.vue'
+import moneyStack from '../../assets/moneyStack.vue'
+import tracking from '../../assets/tracking.vue'
+import arrowUp from '../../assets/arrowUp.vue'
+import arrowDown from '../../assets/arrowDown.vue'
+import BarChart from '../../components/charts/BarChart.vue';
+import RepaymentTable from '../../components/tables/RepaymentTable.vue';
+import queryParam from '../../utilities/queryParam';
+import DatePicker from 'vue2-datepicker'
+import 'vue2-datepicker/index.css'
+import ZeroState from '../../components/ZeroState.vue';
+import BasePagination from '../../components/Pagination/BasePagination.vue';
+export default {
+	components: {
+		DatePicker,
+		StatCard,
+		users,
+		moneyStack,
+		tracking,
+		arrowUp,
+		arrowDown,
+		BarChart,
+		RepaymentTable,
+		BasePagination,
+		ZeroState
+	},
+	data() {
+		return {
+			dateRange: this.getFirstAndLastDayOfCurrentMonth(),
+			locale: "en-US", 
+			dateFormat: "YYYY-MM-DD", 
+			Branches: [],
+			branch: '0',
+			query: {},
+			reports: null,
+			availableRepaymentPlan: [],
+			repaymentPlan: '',
+			apiUrls: {
+				getReports: '/api/repayment-schedule/report',
+				repaymentDurations: '/api/repayment_duration',
+				branches: '/api/branches',
+				renewalList: '/api/renewal/prompters',
+			},
+			pageParams: {},
+			MonthlyStat: [],
+			orders: [],
+			order2: [],
+			isProcessing: true,
+			StatData: [
+				{
+					title: 'Total  Repayment Expected',
+					amount: '',
+					icon: moneyStack
+				},
+				{
+					title: 'Total  Repayment Paid',
+					amount: '',
+					percent: '',
+					percentIcon: arrowUp,
+					icon: tracking
+				},
+				{
+					title: 'Total Defaulters/Total Customers',
+					amount: '',
+					percent: '',
+					percentIcon: arrowDown,
+					icon: users
+				},
+			],
+			barData: {},
+			barOption: {
+				responsive: true,
+				maintainAspectRatio: false
+			}
+		}
+	},
+	methods: {
+		fetchData() {
+			this.pageParams.page = this.$route.query.page
+				? this.$route.query.page
+				: this.pageParams.page;
+			this.pageParams.per_page = this.pageParams.per_page
```

Comment:
Same here:

`this.pageParams.per_page = this.pageParams.per_page || this.$route.query.per_page;`